src/components/Button.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button({ name, color, wide }) {
+  const buttonStyle = {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: '25%',
+    backgroundColor: '#ffa502',
+    margin: 4,
+    fontSize: 20,
+    fontWeight: 'bold',
+    border: '2px solid #000',
+  };
+
+  if (wide) {
+    buttonStyle.width = '50%';
+  }
+  if (color) {
+    buttonStyle.backgroundColor = color;
+  }
+
+  return <div style={buttonStyle}>{name}</div>;
+}
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+  color: PropTypes.string.isRequired,
+  wide: PropTypes.bool.isRequired,
+};
```

Comment:
- Making the `wide` prop as required causes an error since you aren't passing it to all buttons, only the `0` button gets this prop. Kindly fix this.

Reply:
fixed